### PR TITLE
Added two new more AppointmentTypes Vaccination and Surgery

### PR DESCRIPTION
**Pull request by Al-Andrew**

### CODE REVIEW

**Review by OpariucRares — CHANGES_REQUESTED** [review]
nu place

File: src/DucksNet.Domain/Model/Enums/AppoitnmentType.cs [hunk 1]
```diff
@@ -5,6 +5,8 @@ namespace DucksNet.Domain.Model.Enums;
 public class AppointmentType : Enumeration
 {
     public static readonly AppointmentType Consultation = new(1, "Consultation");
+    public static readonly AppointmentType Vaccination = new(2, "Vaccination");
```
Comment (OpariucRares) [inline src/DucksNet.Domain/Model/Enums/AppoitnmentType.cs:8]: misto

File: tests/DucksNet.UnitTests/AppointmentTypeTests.cs [hunk 2]
```diff
@@ -24,7 +24,7 @@ public void When_CreateFromString_WithInvalidAppointmentType_should_Fail()
         result.IsFailure.Should().BeTrue();
         result.Value.Should().BeNull();
         result.Errors.Should().NotBeEmpty();
-        result.Errors.Should().Contain("Invalid AppointmentType string. Valid values are: Consultation.");
+        result.Errors.Should().Contain("Invalid AppointmentType string. Valid values are: Consultation, Vaccination, Surgery.");
```
Comment (OpariucRares) [inline tests/DucksNet.UnitTests/AppointmentTypeTests.cs:27]: nu te cred

File: tests/DucksNet.UnitTests/AppointmentTypeTests.cs [hunk 3]
```diff
@@ -49,6 +49,6 @@ public void When_CreateFromInt_WithInvalidAppointmentType_should_Fail()
         result.IsFailure.Should().BeTrue();
         result.Value.Should().BeNull();
         result.Errors.Should().NotBeEmpty();
-        result.Errors.Should().Contain("Invalid AppointmentType id. Valid values are: 1.");
+        result.Errors.Should().Contain("Invalid AppointmentType id. Valid values are: 1, 2, 3.");
```
Comment (OpariucRares) [inline tests/DucksNet.UnitTests/AppointmentTypeTests.cs:52]: vand golf 4

**Review by OpariucRares — APPROVED** [review]

File: tests/DucksNet.UnitTests/AppointmentTypeTests.cs (hunk 3 above)
Comment (Al-Andrew) [reply tests/DucksNet.UnitTests/AppointmentTypeTests.cs:52]: nu vreau domnu

**Pull request merged**